cmd/build_test.go

```diff
@@ -223,119 +181,3 @@ created: 2009-11-10 23:00:00`,
 		})
 	}
 }
-
-func testBuilderPersistence(t *testing.T, testRegistry string, cmdBuilder func(ClientFactory) *cobra.Command) {
```

Comment:
The implementations of the shared tests are moved into deploy_test.go since it is the superset of deploy+build